MSBuild.lutconfig

```diff
@@ -0,0 +1,7 @@
+<LUTConfig Version="1.0">
+  <Repository />
+  <ParallelBuilds>true</ParallelBuilds>
+  <ParallelTestRuns>true</ParallelTestRuns>
```

Comment:
Wondering about whether this should be false. If something is wrong, it's generally much harder to figure out what it is if there are multiple parallel processes/threads. We miss problems caused by race conditions, but I think that's a reasonable tradeoff. Am I missing something?

Reply:
Can I point you to https://github.com/dotnet/msbuild/pull/7577#discussion_r859018545?